File: public/phase-2-20160620022203/C_SHARP/boy-and-boxes.cs
// Name : Boy and Boxes
using System;
namespace Application {
	class boyand_boxes {
		public static void Main(string[] args) {
		// input:
		// n :  the number of boxes
		// x : number of the
		// a :  number of balls in each box
		int index;
		string[] elements = (Console.ReadLine()).Split(' ');
		int n = Convert.ToInt32(elements[0]);
		int x = Convert.ToInt32(elements[1]);
		int[] a = new int[n];
		string[] a_elements = (Console.ReadLine()).Split(' ');
		for(index=0;index<n;index++)
			a[index] = Convert.ToInt32(a_elements[index]);


		// write your code here
		// store your results in `result`

		// output
		// Dummy Data
		int[] result = new int[n] {3,2,5,4};

		Console.WriteLine(string.Join(" ", result));
		}	
	}
}

File: public/phase-2-20160620022203/C_PLUS_PLUS/boy-and-boxes.cpp
// Name : Boy and Boxes
#include <iostream>
#include <string>
using namespace std;

int main(int argc, char const *argv[]) {
	// input:
	// n :  the number of boxes
	// x : number of the
	// a :  number of balls in each box
	int index;
	int n, x;

	cin >> n >> x;
	int  a[n];

	for(index=0;index<n;index++)
		cin >> a[index];


	// write your code here
	// store your results in `result`

	// output
	// Dummy Data
	int  result[] = {3,2,5,4};

	for(index=0;index<n;index++)
		cout << result[index] << " ";

}